test/test_respondent_controller.py

```diff
@@ -1215,6 +1215,30 @@ def test_put_email_verification_uses_case_insensitive_email_query(self):
             account_controller.put_email_verification(token)
             query.assert_called_once_with("test@example.test", db.session())
 
+    def test_email_verification_resets_password_reset(self):
```

Comment:
I struggle to understand this test from the name, ```test_token_removed_on_email_update``` maybe, something like that. If it's to difficult to explain consider putting in Given, When, Then.

Reply:
test name has been changed